gpu_ocean/SWESimulators/CTCS.py

```diff
@@ -194,9 +194,12 @@ def fromfilename(cls, gpu_ctx, filename, cont_write_netcdf=True):
         timeIntegrator = sim_reader.get("time_integrator")
         y_zero_reference_cell = sim_reader.get("y_zero_reference_cell")
 
-        wind_stress_type = sim_reader.get("wind_stress_type")
-        wind = Common.WindStressParams(type=wind_stress_type)
-
+        try:
```

Comment:
Shouldn't the sim_writer always write out the wind_stress_type?

Reply:
SimWriter and SimReader are not tested in this PR, as it needs non-trivial refactoring for the new wind stresses. Should we write all the wind fields to the NetCDF? Should we just refer to the original NetCDF, or make a new NetCDF if necessary and refer to that one?

I suggest to fix this issue in another PR. Added card in Trello: https://trello.com/c/fzDQPkwd

Reply:
I think it makes most sense to refer to the forcing input file(s) in global attributes. If an output file is used to hotstart a simulation, the forcing for already simulated time-steps is not very useful anyway.